gempython/tools/xdaq/setDAQParamsPostConfigure.py

```diff
@@ -3,24 +3,90 @@
 from gempython.tools.amc_user_functions_uhal import *
 from gempython.utils.gemlogger import printGreen, printRed, printYellow
 
+def convert2Int(inputStr):
```

Comment:
Generally not a fan of attempts at a generic wrapper to parse a `string` as an `int` because there are plenty of corner cases that should be covered
I prefer that when an argument can be represented in a variety of ways, this should be defined in the usage, and an expectation mentioned there.
Some of the problems here are that:
* `0xbad` is identical to `bad`, but `bad` will be parsed as a binary and will fail.
* Similarly, `0xcafe` is identical to `cafe`, which would be parsed as a decimal and will fail.

Reply:
removed this function